FlowCrypt/Controllers/Setup/SetupManuallyEnterPassPhraseViewController.swift

```diff
@@ -244,21 +244,17 @@ extension SetupManuallyEnterPassPhraseViewController {
         )
 
         if storageMethod == .memory {
-            try keysToUpdate
-                .map {
-                    PassPhrase(value: passPhrase, fingerprintsOfAssociatedKey: $0.fingerprints)
-                }
-                .forEach {
-                    try appContext.passPhraseService.updatePassPhrase(with: $0, storageMethod: storageMethod)
-                }
+            for newPassPhrase in (keysToUpdate.map {
+                PassPhrase(value: passPhrase, fingerprintsOfAssociatedKey: $0.fingerprints)
+            }) {
```

Comment:
If it reasonably fits on one line, then let's define it as:

```swift
for newPassPhrase in keysToUpdate.map { PassPhrase(value: passPhrase, fingerprintsOfAssociatedKey: $0.fingerprints) } {
  // ...
}
```

If it doesn't, then let's avoid multiline `for-in` header. In that case, let's define a variable.

```swift
let newPassPhrases = keysToUpdate.map { PassPhrase(value: passPhrase, fingerprintsOfAssociatedKey: $0.fingerprints) }
for newPassPhrase in  {
  // ...
}
```

In this particular case though, we could I guess simplify a bit as follows: add an overload `PassPhrase` init as follows: `init(value: passPhrase, associatedKey: Key)` (or similar)

Then you could comfortably on one line:

```swift
for newPassPhrase in keysToUpdate.map { PassPhrase(value: passPhrase, associatedKey: $0) } {
  // ...
}
```

Reply:
The PR looks good overall, just need to figure out the exact style - a multi-line for-in definition doesn't read well:

```swift
for x in (y.map {
  // ...
}) {

}
```

These two options read better:

```swift
val x = abc.map { $0.d }
for y in x {
  zz
}
```

or 

```swift
for y in abc.map { $0.d } {
  zz
}
```

Reply:
I agree.
I will choose this style because if we put in one line, line length will be super long for some cases (which is not good for reading)
```
let x = abc.map { $0.d }
for y in x {
  zz
}

```

Reply:
consistency is good. Looks good to me